### PR TITLE
ADD HEADER, STROKE, FLEXBOXES

### PR DESCRIPTION
**Pull request by Hashlemon**
Done on Flexes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,10 +2,82 @@
 <html lang="ru-BLR">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>What do we offer</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+<style>
```
Comment (romos21) [inline index.html:8]: @Hashlemon it's better to transfer styles into css file

Comment (romos21) [reply index.html:8]: @Hashlemon Also. As I remember right, Roboto is an inbuilt font

Comment (Hashlemon) [reply index.html]: Done. 
Roboto is not inbuilt in Ubuntu at least. So I leave this font taken outside

**Comment by romos21** [comment]
@Hashlemon done!

**Pull request merged**